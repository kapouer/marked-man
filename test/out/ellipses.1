ELLIPSES(7)                                                        ELLIPSES(7)

NNAAMMEE
       eelllliippsseess - testing ellipses

       Ellipses should be replaced...

       ...also, they shouldn't interfere with regular dots at the beginning
       of a line.

                                 January 1979                      ELLIPSES(7)
