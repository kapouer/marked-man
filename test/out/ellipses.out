[4mELLIPSES[24m(7)            Miscellaneous Information Manual            [4mELLIPSES[24m(7)

[1mNAME[0m
       [1mellipses [22m- testing ellipses

       Ellipses should be replaced...

       ...also, they shouldnâ€™t interfere with regular dots at the beginning
       of a line.

                                 January 1979                      [4mELLIPSES[24m(7)
